docs/edge/References.md.hbs

```diff
@@ -8,14 +8,15 @@ By themselves, references are useless - they must *resolve* to a *{{{createLink
 The resolution algorithm looks like this:
 
 1. Is the reference a special ref? If so, resolve with the appropriate special keypath.
-2. If the reference is explicit or matches a path in the current context exactly, resolve with that keypath.
-3. Grab the current virtual node from the template hierarchy.
-4. If there are any aliases, index, or key mapping defined that match, resolve with that keypath.
-5. If this is a component and there are any matching mappings, resolve with that keypath.
-6. If this node has context and there is a matching path, resolve with that keypath.
-7. Otherwise, remove the innermost context from the stack. Repeat (3-7).
-8. See if `reference` is a valid keypath by itself. If so, resolve with that keypath.
-9. If the reference is still unresolved, add it to the 'pending resolution' pile. Each time potentially matching keypaths are updated, resolution will be attempted for the unresolved reference.
+2. If the reference is prefixed (`./`, `../`, `^^/`), resolve it.
+3. If the current context has a matching key, then resolve with it.
+4. Grab the current virtual node from the template hierarchy.
+5. If there are any aliases, index, or key mapping defined that match, resolve with that keypath.
+6. If this is a component and there are any matching mappings, resolve with that keypath.
+7. If this node has context and there is a matching path, resolve with that keypath.
+8. Otherwise, remove the innermost context from the stack. Repeat (3-7).
+9. If not disabled, check to see if there's a matching Ractive instance method or property, and if so, resolve with it.
```

Comment:
I am not sure what is disabled/enabled here. First thing that comes into mind is `isolated` but doesn't fit the bill.